### DISCUSSION TITLE
排序出现问题

### DISCUSSION THREAD

**Discussion by sdqOS** [walinejs/waline#1381] [Q&A] [answered]
尊敬的作者，你好

@waline/client@2.7.0 和 @waline/ercel@1.20.0 我都安装了，

并没有出现我想要的结果

我的版本是"@waline/vercel": "^1.20.1"  

请问一下该问题的解决方法

**Comment by lizheming** [comment, marked as answer]
提供一下可复现问题的地址我看一下

**Reply by sdqOS** [reply]
https://sdq3.ml/valine.html

**Reply by lizheming** [reply]
服务端版本不对，现在是 1.19.4 的版本，如何更新见文档 https://waline.js.org/advanced/faq.html#%E5%A6%82%E4%BD%95%E5%8D%87%E7%BA%A7
